manifests/c/com/malawarecreator/neofile/0.0.0.1/com.malawarecreator.neofile.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: com.malawarecreator.neofile

```

```suggestion
PackageIdentifier: malwarecreator.neofile
```
The preferred identifier is `Publisher.Package`. After updating this, the folder the manifest is in will need to be changed